README.md

```diff
@@ -1104,6 +1104,7 @@ API | Description | Auth | HTTPS | CORS |
 | [RandomUser](https://randomuser.me) | Generates and list user data | No | Yes | Unknown |
 | [RoboHash](https://robohash.org/) | Generate random robot/alien avatars | No | Yes | Unknown |
 | [Spanish random words](https://palabras-aleatorias-public-api.herokuapp.com) | Generate spanish words randomly | No | Yes | Unknown |
+| [Spanish random names](https://random-names-api.herokuapp.com) | Generate spanish names (with gender) randomly | No | Yes | Unknown |
```

Comment:
Fix the alphabetical order (before `Spanish random words`) and update to correct link, please:

```suggestion
| [Spanish random names](https://random-names-api.herokuapp.com/public) | Generate spanish names (with gender) randomly | No | Yes | Unknown |
```